### PR TITLE
Update

### PR DESCRIPTION
**Pull request by EnnoMeijers**
Updated the core and config to match the latest LDWizard Core version and rewrote the Termennetwerk Refinement so it supports all the sources in the Network of Terms.

### CODE REVIEW

File: termennetwerk.ts [hunk 1]
```diff
@@ -1,24 +1,38 @@
...
-interface GraphQlResponse {
+interface Term {
+  uri: string;
+  prefLabel: string[];
+  altLabel: string[];
+}
+
+interface Result {
+  __typename: string;
+  terms: Term[];
```
Comment (ddeboer) [inline termennetwerk.ts:14]: This does not seem to account for error conditions. If the terminology source returns an error, the GraphQL response [looks different](https://github.com/netwerk-digitaal-erfgoed/network-of-terms/tree/master/packages/network-of-terms-graphql#query-a-single-source).

File: termennetwerk.ts [hunk 2]
```diff
@@ -45,29 +59,93 @@ const graphQlQuery = gql`
...
+function selectTerm(result: Result, searchTerm: string, source: string): Term | undefined {
+
+    if (result.__typename === 'ServerError') { // catch server errors
+      //console.log("Termennetwerk refinement: server error for source",source);
+      return undefined; 
+    }
+
     if (result.__typename === 'TimeoutError') { // catch timeouts errors
-      console.error(`Timeout error from Network-of-Terms API: "${result.message}" for keyword: "${searchTerm}"`);
+      //console.log("Termennetwerk refinement: time out for source", source);
```
Comment (ddeboer) [inline termennetwerk.ts]: Why remove the logging?

**Review by ddeboer — CHANGES_REQUESTED** [review]

File: termennetwerk.ts (hunk 1 above)
Comment (EnnoMeijers) [reply termennetwerk.ts:14]: Good point - I will add the error conditions, do you have a pointer to code I can reuse?

Comment (ddeboer) [reply termennetwerk.ts:14]: Perhaps the demonstrator code is a good starting point: https://github.com/netwerk-digitaal-erfgoed/network-of-terms-demo/blob/4d03e014698c4605b9c03f60ccea072fd0d6abdd/src/query.ts#L5

Comment (EnnoMeijers) [reply termennetwerk.ts:14]: Thanks, that is helpful! Should we consider making these types part of the network-of-terms package so there is no need to define these every time, or is this too client specific?

Comment (EnnoMeijers) [reply termennetwerk.ts:14]: On second thought: the code checks for error conditions through the `__typename` contents before the Term results are addressed, so I prefer to keep current interface definitions for the sake of complexity.

File: termennetwerk.ts (hunk 2 above)
Comment (EnnoMeijers) [reply termennetwerk.ts]: To avoid too much noise, but your right I activated them again.

**Pull request merged**